app/services/file_manager/file_list.py

```diff
@@ -23,35 +25,48 @@ def __init__(self):
         super().__init__(AppConfig.Connections.url_bff)
 
         self.endpoint = AppConfig.Connections.url_bff + '/v1'
+        self.zone_map = {
+            0: AppConfig.Env.green_zone,
+            1: AppConfig.Env.core_zone,
+        }
```

Comment:
I think we should place this logic in app config to make it easier to maintain. We did this in bff-web here: https://github.com/PilotDataPlatform/bff-web/blob/develop/app/config.py#L147-L150

Reply:
thanks will update it!

Reply:
Thank you!